### PR TITLE
Add `max-width` to usage data settings

### PR DESCRIPTION
**Pull request by manuthecoder**
See #6480

### CODE REVIEW

**Review by sampoder — COMMENTED** [review]
I think we should set this on `field--options`, thoughts @garyhtou?

**Comment by garyhtou** [comment]
> I think we should set this on `field--options`, thoughts @garyhtou?

I'll leave that decision up to you and Manu!

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -171,7 +171,7 @@
             Off by default.
           <% end %>
         </p>
-        <div class="field field--options">
+        <div class="field field--options" style="max-width: 420px">
```
Comment (sarthaktexas) [inline app/views/users/edit.html.erb]: why are we using pixels here?

Comment (sarthaktexas) [reply app/views/users/edit.html.erb]: we can keep it consistent by using more responsive viewport units

Comment (sarthaktexas) [reply app/views/users/edit.html.erb]: also, i agree with sam that this should be added to `field--options` instead, that would be a cleaner solution

**Review by sarthaktexas — DISMISSED** [review]

File: app/views/users/edit.html.erb (hunk 1 above)
Comment (manu-gurudath) [reply app/views/users/edit.html.erb]: I kept it the same as the previous section - I could change both of the section's widths (see line 144)

**Review by sampoder — APPROVED** [review]
Cool! Lets do it.

**Pull request merged**